app/models/blueprint.rb

```diff
@@ -1,4 +1,6 @@
 class Blueprint < ApplicationRecord
+  has_many :versions, :class_name => 'Blueprint',
+           :foreign_key => :original_blueprint_id, :primary_key => :original_blueprint_id
```

Comment:
This is interesting as it allows you to fetch the entire blueprint lineage in one query, but there's no way to know the "parent" version, nor be able to show the transitions over time (I guess you could use created_at, but that feels icky in some way to me).  I'm wondering if this would be a lot cleaner with an `ancestry` based implementation.  That way you ask questions like `parent`, `children`, `subtree`, etc.  Additionally, if the leaf node is always active, then `active_version?` can we written as `is_childless?`, and you don't need a column for it

@kbrock Thoughts?